api/types/src/transaction.rs

```diff
@@ -665,6 +670,44 @@ impl TryFrom<Script> for ScriptPayload {
     }
 }
 
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize, Object)]
+pub struct MultisigTransactionPayload {
+    pub function: EntryFunctionId,
+    /// Type arguments of the function
+    pub type_arguments: Vec<MoveType>,
+    /// Arguments of the function
+    pub arguments: Vec<serde_json::Value>,
+}
+
+/// A multisig transaction that allows an owner of a multisig account to execute a pre-approved
+/// transaction as the multisig account.
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize, Object)]
+pub struct MultisigPayload {
+    pub multisig_address: Address,
+    pub transaction_id: u64,
+
+    // Transaction payload is optional if already stored on chain.
+    #[oai(skip)]
```

Comment:
what's this for?

Reply:
This was removed. I thought this has the same effect as skip_if=Option::none like for serde, but it's totally not.